src/mscorlib/shared/System/Nullable.cs

```diff
@@ -14,10 +14,10 @@ namespace System
     [Serializable]
     [NonVersionable] // This only applies to field layout
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    public readonly struct Nullable<T> where T : struct
+    public struct Nullable<T> where T : struct
     {
         private readonly bool hasValue; // Do not rename (binary serialization)
-        internal readonly T value; // Do not rename (binary serialization)
+        internal T value; // Do not rename (binary serialization)
```

Comment:
Add a comment why this cannot be readonly?

Reply:
Will add. The test I've added in corefx will also fail if this is made read-only.